docs/fastdds/dds_layer/publisher/publisher/publisher.rst

```diff
@@ -3,3 +3,84 @@
 Publisher
 =========
 
+The :class:`Publisher` acts on behalf of one or several :ref:`dds_layer_publisher_dataWriter` objects
+that belong to it.
+It serves as a container that allows grouping different :ref:`dds_layer_publisher_dataWriter` objects under
+a common configuration given by the :class:`PublisherQos` of the :class:`Publisher`.
+
+:ref:`dds_layer_publisher_dataWriter` objects that belong to the same :class:`Publisher` do not have any other
+relation among each other beyond the :class:`PublisherQos` of the :class:`Publisher` and act independently
+otherwise.
+Specifically, a :class:`Publisher` can host :ref:`dds_layer_publisher_dataWriter` objects for different topics
+and data types.
+
+
+.. _dds_layer_publisher_publisherQos:
+
+PublisherQos
+------------
+
+:class:`PublisherQos` controls the behavior of the :ref:`dds_layer_publisher_publisher`.
+Internally it contains the following :class:`QosPolicy` objects:
+
++--------------------------------+------------------------------------+----------+
+| QosPolicy class                | Accessor/Mutator                   | Mutable  |
++================================+====================================+==========+
+| PresentationQosPolicy          | :func:`presentation`               | Yes      |
++--------------------------------+------------------------------------+----------+
+| PartitionQosPolicy             | :func:`partition`                  | Yes      |
++--------------------------------+------------------------------------+----------+
+| GroupDataQosPolicy             | :func:`group_data`                 | Yes      |
++--------------------------------+------------------------------------+----------+
+| EntityFactoryQosPolicy         | :func:`entity_factory`             | Yes      |
++--------------------------------+------------------------------------+----------+
+
+Refer to the detailed description of each :class:`QosPolicy` class for more information about their usage and
+default values.
+
+The QoS value of a previously created :ref:`dds_layer_publisher_publisher` can be modified using the
+:func:`set_qos` member function.
+
+.. literalinclude:: /../code/DDSCodeTester.cpp
+   :language: c++
+   :start-after: //DDS_CHANGE_PUBLISHERQOS
+   :end-before: //!
+   :dedent: 8
+
+
+.. _dds_layer_defaultPublisherQos:
+
+Default PublisherQos
+^^^^^^^^^^^^^^^^^^^^
+
+The default :ref:`dds_layer_publisher_publisherQos` refers to the value returned by the
+:func:`get_default_publisher_qos` member function on the :ref:`dds_layer_domainParticipant` instance.
+The special value ``PUBLISHER_QOS_DEFAULT`` can be used as QoS argument on :func:`create_publisher`
+or :func:`set_qos` member functions to indicate that the current default :ref:`dds_layer_publisher_publisherQos`
```

Comment:
Is it to use the current default or the constructed default? Below it says constructed